recipe/meta.yaml

```diff
@@ -1,79 +1,84 @@
 {% set name = "langchain" %}
-{% set version = "0.0.157" %}
+{% set version = "0.0.298" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/langchain-{{ version }}.tar.gz
-  sha256: e93411664b09988ad100c807ff3f8eb118e053d926878fd4e3cbaa0f2279a467
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/langchain-{{ version }}.tar.gz
+    sha256: d8425a05aee5dc5474d2a1fae766b7e19b5b08d8facac2d4a81353ce3df80322
+  - url: https://github.com/langchain-ai/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: b24039599f2ba23047b1aacaeb656361854d813d4d35f6088d2a7cfb7f809f6b
+    folder: github-source
 
 build:
+  number: 0
   skip: True # [py<38]
+  script: 
+    - cp github-source/LICENSE LICENSE
```

Comment:
this should be done in the about section I think. Unless it's the unpacking of the second tarball that causes issues.

Reply:
The license is missing in the source dist on PyPi. Conda-forge for some reason also has 2 source/url

Reply:
So that license file is coming from the second tarball, is that right?

Reply:
Yep